mods/default/locale/template.txt

```diff
@@ -135,7 +135,13 @@ Glass=
 Obsidian Glass=
 Brick Block=
 Mese Lamp=
-Mese Post Light=
+
+Apple Wood Mese Post Light=
```

Comment:
there's no need to update the template files, that will be done by us before the release